docs/admin/securing_workstation.rst

```diff
@@ -2,3 +2,58 @@ Keeping the Workstation secure
 ==============================
 
 .. include:: ../includes/top-warning.rst
+
+The *SecureDrop Workstation* provides the combined functionality of the 
+Tails-based *Journalist Workstation* and *Secure Viewing Station* (SVS). As such,
+it contains both a copy of the submission private key, and encrypted and 
```

Comment:
If we're sticking with standard docs formatting , `*Submission Private Key*`, but also a nit.

Reply:
:heavy_check_mark:

Reply:
:heavy_check_mark: